fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/rdf/RdfStream.java

```diff
@@ -13,12 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package org.fcrepo.kernel.api.rdf;
```

Comment:
Why `api.rdf`? Why not just `org.fcrepo.kernel.api`? It's one of the most fundamental types we use.